frontend/src/assets/css/variables.scss

```diff
@@ -63,7 +63,7 @@
     --error-primary: var(--torch-red);
 
     // Header
-    --header-height: 60px;
+    --header-height: 72px;
```

Comment:
we don't broke something with this change?

Reply:
definitely not, I contacted with @Simpsonkk and he tested new value. and according page layout styles also was changed frontend/src/bundles/common/components/page-layout/styles.module.scss